config/pipeline.yaml

```diff
@@ -3803,6 +3806,10 @@ build_configs:
     tree: net-next
     branch: 'main'
 
+  netdev-testing:
+    tree: netdev-testing
+    branch: 'https://netdev.bots.linux.dev/static/nipa/branches.json'
```

Comment:
The branches we are going to use comes from this file that Jakub just put together. They will only include code that  pass SW tests and will be generated every 8h usually.
 https://netdev.bots.linux.dev/static/nipa/branches-hw.json

Reply:
I will update this after merging

Reply:
Fixed in https://github.com/kernelci/kernelci-pipeline/pull/1019